dev/com.ibm.ws.jaxrs.2.0.client/src/com/ibm/ws/jaxrs20/client/security/LibertyJaxRsClientSSLOutInterceptor.java

```diff
@@ -68,7 +68,7 @@ public void handleMessage(Message message) throws Fault {
             return; // only process HTTPS requests
         }
 
-        if (!overrideUserTLS && PropertyUtils.isTrue(message.get("com.ibm.ws.jaxrs.client.security.skipSSLConfigInterceptor"))) {
```

Comment:
is this going to affect any customers who might be setting this property? Or do we not expose this property and we're fine to just use CXF's property?

Reply:
> is this going to affect any customers who might be setting this property? Or do we not expose this property and we're fine to just use CXF's property?

The latter.  We don't expose this property - it is only used internally for cases where the user has invoked an SSL-related API in either the JAX-RS Client or MP Rest Client - then we set that property so that we override the normal SSL config process with the setting specified by the user.